frontend/pages/queries/QueryPage/QueryPage.jsx

```diff
@@ -83,9 +84,9 @@ export class QueryPage extends Component {
   }
 
   componentWillMount () {
-    const { dispatch, hostIDs, selectedHosts, selectedTargets } = this.props;
+    const { dispatch, hostIDs, hostUUIDs, selectedHosts, selectedTargets } = this.props;
 
-    if (hostIDs) {
+    if (hostIDs || hostUUIDs) {
```

Comment:
Should we verify if it's an array that is has data?

Reply:
Unfortunately (IMO) the JS API gives us a string if a single parameter was provided, or an array of strings if multiple parameters were provided. Do you think there's a good way to check for that?